src/convertToObject.js

```diff
@@ -6,7 +6,69 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const rawLines = sourceString.split(';');
+
+  // filter lines without letters
+  const noEmptyLines = rawLines
+    .map((line) => {
+      return line.trim();
+    })
+    .filter((item) => item.length > 0);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. Align `.map` and `.filter` with `.split` on line 13.